content/docs/setup/kubernetes/spec-requirements/index.md

```diff
@@ -38,3 +38,47 @@ cluster must satisfy the following requirements:
 
 * _**`NET_ADMIN` capability**:_ If pod security policies are enforced in your cluster and unless you use the [Istio CNI Plugin](/docs/setup/kubernetes/istio-cni-install/), your pods must have the `NET_ADMIN` capability allowed.
 See [Required Pod Capabilities](/help/ops/setup/required-pod-capabilities/).
+
+* _**Service Ports and Protocols**_: Ensure that your service's ports and protocols are available and don't conflict with the [Istio used ports and protocols](#istio-used-ports) for the Istio components be used.
+
+## Istio Used Ports
+
+The following ports and protocols are used by Istio.
+
+If Istio is using HTTP on a port, a service may also use HTTP, but not TCP on that port. This is because HTTP routing is dependent on a number of factors such as Hostname, headers or path which removes the conflict.
+
+If Istio is using TCP, another service may not use HTTP on that port. The service may potentially use TCP on that port, marked with a `?` in the table, by using SNI headers or IP address blocks. For example, see [Consuming External TCP Services](/blog/2018/egress-tcp/).
+
```

Comment:
neither of the above two are true. I understand the Kafka issue but the warnings here do nothing to address that.
Simply state that do not have any TCP headless services on the same port as Istio's TCP ports. Thats it

Reply:
Done.